docs/cip/CIP-12062023_System_Info.md

```diff
@@ -0,0 +1,214 @@
+# CIP-7: System Info
+
+## Status
+
+Current Status: `Under Discussion`
+
+## Motivation
+
+Currently, a lot of the support discussions in Discord revolve around gathering information about the user's operating
+environment. This information is crucial for debugging and troubleshooting issues. We want to make it easier for users
+to provide this information.
+
+## Public Interfaces
+
+This proposal introduces a new `API` endpoint `/api/v1/env` that will return a dictionary with system information based
+on flags provided by the user.
+
+The endpoint will accept the following flags:
+
+- `python_version` - Shows Python related information. Possible values `True` or `False` (default `True`)
```

Comment:
Why are we making these conditional? I think its fine to always return them.

Reply:
The reasoning is in line with `env rstat` and potentially other similar commands for streaming observability data from the server where we don't need to generate and return all the information.

Reply:
I agree with hammad that we can always return this. Depending on how `env rstat` is implemented we can either skip returning python version in the response for it or do something else.

For that matter, I think _all_ of these can be default true: `/api/v1/env` can just return a dict with a bunch of env info. IMO configurability just adds unneeded complexity right now.